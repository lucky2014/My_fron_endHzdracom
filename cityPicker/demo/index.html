<!doctype HTML>
<html>
<head>
	<meta charset="utf-8"/>
	<title>前端开发-杭州龙骞科技前端部门</title>
	<script src="../../js/jquery/jquery-1.8.3.min.js"></script>
	<link type="text/css" rel="stylesheet" href="css/cityPicker.css" />
</head>
<body style="margin:0; padding:0;">
	<!--城市选择插件-->
	<div id="cityPicker">
		<div>
            <p><i data-num="">请选择省份</i><em class="arrow"></em></p>
            <ul class="hide">
            </ul>
        </div>
        <div>
            <p><i data-num="">请选择城市</i><em class="arrow"></em></p>
            <ul class="hide">
            </ul>
        </div>
        <div>
            <p><i data-num="">请选择县</i><em class="arrow"></em></p>
            <ul class="hide">
            </ul>
        </div>
    </div>

<script src="js/cityPicker.js"></script>
<script>
$(function(){
    $("#cityPicker").cityPicker({
        url: "js/cityData.min.js",
        hasCountry: true
    });
});
</script>
</body>
</html>