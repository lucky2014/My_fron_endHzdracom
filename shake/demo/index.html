<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=320,initial-scale=1.0,maximum-scale=1.0,user-scalable=0">
	<meta name="apple-mobile-web-app-capable" content="yes">
	<meta name="format-detection" content="telephone=no">
	<meta charset="utf-8">
	<title>摇一摇</title>
	<script src="js/jquery-1.8.3.min.js"></script>
	<script src="js/shake.js"></script>
</head>
<body>
	<script>
		window.onload = function() { 
			var myShakeEvent = new Shake({
				threshold:15
			});
			myShakeEvent.start();
			window.addEventListener('shake', shakeEventDidOccur, false);
			function shakeEventDidOccur () {
				alert("摇动了")      //手机摇动是要触发的事件或要执行的函数
			}
		};
	</script>
</body>
</html>