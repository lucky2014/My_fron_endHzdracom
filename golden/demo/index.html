<html>
	<head>
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=320,initial-scale=1.0,maximum-scale=1.0,user-scalable=0">
		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="format-detection" content="telephone=no">
		<meta charset="utf-8"/>
		<title>砸金蛋活动模板</title>
		<link rel="stylesheet" href="css/smashing.css">
		<script src="js/jquery-1.8.3.min.js"></script>
		<script src="js/jquery.touchSwipe.js"></script>
		<script src="js/smashing.js"></script>
	</head>
	<body>
				<div class="box main_box">
					<ul data-type="true" data-animate="false">
						<li class="img1">
							<img src="images/egg.png" alt="">
						</li>
						<li class="img2">
							<img src="images/egg.png" alt="">
						</li>
						<li class="img3">
							<img src="images/egg.png" alt="">
						</li>
						<li class="img4">
							<img src="images/egg.png" alt="">
						</li>
					</ul>
					<span class="hammer"><img src="images/hammer.png" alt=""></span>
				</div>
				<script>
					var options = {
						imgArr:["images/egg.png","images/eggBreak.png"],
						callback:function(){
							alert("蛋砸碎了")
						}
				    };
					$(".main_box").smashing(options);
				</script>
	</body>
</html>